### ISSUE TITLE
Local Notification direction arrows are always right-oriented

### ISSUE THREAD

**Issue by billyking991** [mapbox/mapbox-navigation-ios#2267]
I've been testing Local Notifications for about a week and I'm only seeing right-oriented arrows in the notifications.  If I come up to a left turn, the arrow displayed in the notification is a right arrow.  This also goes for left merges, when the destination is on the left, and any other left-oriented directions.  I have yet to see any left turns in the notifications.

My implementation consists of only adding the UserNotifications framework and asking for permission to display local notifications:

let center = UNUserNotificationCenter.current()
        center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
            // Enable or disable features based on authorization.
        }

I was told by QA that this is a known issue and was asked to submit this issue.  Please let me know if you need any additional info from me.  I don't have screenshots at this time, but I can get them if you want to see anything specific.

**Mapbox Navigation SDK version:**
0.38.0

**Comment by 1ec5** [comment]
Were the images oriented correctly in a previous version? #2206 added these icons based on factoring out the maneuver icon generation code used in CarPlay. That code calls `UIImage.withHorizontallyFlippedOrientation()`, which affects the `UIImage.imageOrientation` property but doesn’t affect the underlying pixel data. Maybe the lock screen isn’t honoring the `UIImage.imageOrientation` property?

**Comment by billyking991** [comment]
I only recently added in notifications to my app, so I haven't tried it out on any other Mapbox SDK version.  I've only tried it on 0.38.0.  The images in the NavView are fine.  It is the images in the notification that aren't correct (for left turns).  I've only tested this on iOS 13 and don't have any devices still on iOS 12.

Is there a way I can check during testing to see if the lock screen isn't honoring the imageOrientation property.  I'd be happy to run some tests tomorrow if you tell me what to look for...
